### ISSUE TITLE
[Static web assets] Add section detailing third-party integration with static web assets

### ISSUE THREAD

**Issue by javiercn** [dotnet/AspNetCore.Docs#24808]
### TL;DR

Items must be declared in the `Content` item group before the `AssignTargetPaths` target.

For integrating with JS build processes, reference the client assets package.

More details can be found https://devblogs.microsoft.com/dotnet/build-client-web-assets-for-your-razor-class-library/

This is .NET 6.0 and onwards.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9fd019c7-ceb7-6ef3-26ba-6b44341b6fc5
* Version Independent ID: fd116d00-0dc7-efaf-c714-34d77b345a05
* Content: [Reusable Razor UI in class libraries with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/razor-pages/ui-class.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/razor-pages/ui-class.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-razorpages**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by SteveSandersonMS** [comment]
Since the issue description is fairly ad-hoc, just want to make sure the original goal is preserved:

> We need to decide which parts of SWA are public, supported APIs (e.g., which MSBuild tasks, properties, metadata, enum values, etc.) and ensure they are fully documented and remain consistent in behavior across versions.
